### DISCUSSION TITLE
"Non-shared" axes

### DISCUSSION THREAD

**Discussion by CodeWithMichal** [Live-Charts/LiveCharts2#218] [Q&A] [answered]
Hi,

Currently there is possibility to add many axes to chart but they occupy the same space. Is there any possibility to create axes which divide chart area into independent pieces? Something like below:
https://image.shutterstock.com/image-vector/brain-wave-frequencies-on-black-260nw-1316060591.jpg

@beto-rodriguez is it possible to implement something like that with current version?

**Comment by beto-rodriguez** [comment]
What about creating multiple charts?

**Comment by CodeWithMichal** [comment]
Its all about space. Multiple charts occupy too much space and its quite critical part of my application

**Comment by beto-rodriguez** [comment, marked as answer]
Try reducing the `DataPadding` and making the X axis invisible:

``` c#
Series = new ISeries[]
{
    new StepLineSeries<int>
    {
        // disable the data padding on the Y axis
        // https://github.com/beto-rodriguez/LiveCharts2/blob/master/docs/components/1.8.series.md#datapadding
        DataPadding = new LvcPoint(0.5f, 0),
        Values = values
    }
};

X = new Axis[]
{
    new Axis
    {
        IsVisible = false
    }
};
```
